doc/api_assets/style.css

```diff
@@ -488,3 +488,21 @@ th > *:last-child, td > *:last-child {
     font-size: 3.5em;
   }
 }
+
+
+@media print {
+
+  html  {
```

Comment:
Nit: unnecessary double spaces after `html`, use a single space between `html` and `{`.